metaci/build/management/commands/queue_build.py

```diff
@@ -0,0 +1,37 @@
+from django.core.management.base import BaseCommand
+
+from metaci.build.management.commands._utils import make_build
+
+
+class Command(BaseCommand):
```

Comment:
@prescod - You mentioned maybe delete this?

Reply:
04d6c070